composer.json

```diff
@@ -16,7 +16,7 @@
 		}
 	},
 	"replace": {
-		"rte_ckeditor_image": "self.version",
```

Comment:
What makes this obsolete? Is this change unrelated to renaming the package?
I thought this is a TYPO3 requirement?